src/accountsdb/db.zig

```diff
@@ -81,24 +81,25 @@ pub const AccountsDB = struct {
             config.storage_cache_size,
         );
 
-        var disk_allocator_ptr: ?*DiskMemoryAllocator = null;
-        var reference_allocator = std.heap.page_allocator;
-        if (config.disk_index_path) |disk_index_path| {
-            var ptr = try allocator.create(DiskMemoryAllocator);
+        const maybe_disk_allocator: ?*DiskMemoryAllocator, //
+        const reference_allocator: std.mem.Allocator //
```

Comment:
Do you mean `if (config.disk_index_path) blk: {...} else blk: {...}`?

Reply:
cc8145229a34502d3b32a84672d7d17f89bdc9b6

Reply:
i meant inside the main blk you originally had:
```
const a, const b = blk: {
    // logic here
}
```
the pros are you get 1) symmetry in the if/else with both `break :blk` and 2) only use a single `blk:`

Reply:
That seems worse to me, you're adding another unnecessary level of indentation. The way I did it originally used an early return style specifically for that reason and also because of the single `blk:`.

Reply:
makes sense - wdyt about changing `else` to `else blk:` - i feel like code symmetry leads to clear code flow

Reply:
Sorry, but symmetry with what, or in what way exactly?

Reply:
the logic in both the if and else branch is the same with a clear `break blk: value`
this
```
   const x = blk: {
            if (true) {
                break :blk 4;
            } else {
                break :blk 3;
            }
    }
```
vs this
```
            const x = if (true) blk: {
                break :blk 4;
            } else 3;
```

Reply:
e727b2249d8bb0b49b218d324e27f8b6a2543ee3

Reply:
I'm not a huge fan, but I see your point